src/index.coffee

```diff
@@ -132,6 +132,68 @@
           window._hmt.push(args)
       }
 
+    mixpanel: (account) ->
+      return unless account
+      lib_name = 'mixpanel';
+      window.mixpanel = [];
+      mixpanel._i = [];
+
+      mixpanel.init = (token, config, name) ->
+        # support multiple mixpanel instances
+        target = mixpanel
+        if name?
+          target = mixpanel[name] = []
+        else
+          name = lib_name;
+
+        # Pass in current people object if it exists
+        target.people or= []
+
+        target.toString = (no_stub) ->
+          str = lib_name
+          str += '.' + name if name isnt lib_name
+          str += ' (stub)' unless no_stub
+          return str
+
+        target.people.toString = ->
+          target.toString(1) + '.people (stub)'
+
+        _set_and_defer = (target, fn) ->
+          split = fn.split('.')
+          if split.length is 2
+            target = target[split[0]]
+            fn = split[1]
+
+          target[fn] = ->
+            target.push([fn].concat(slice.call(arguments)))
+
+        functions = 'disable track track_pageview track_links track_forms register register_once alias unregister identify name_tag set_config people.set people.set_once people.increment people.append people.track_charge people.clear_charges people.delete_user'.split(' ')
```

Comment:
You should define these methods as an Array just like:
>
https://github.com/teambition/gta/blob/master/src/index.coffee#L179

It would be more clear in format and improve a little performance.